### ISSUE TITLE
how to save sc$pl$umap figure

### ISSUE THREAD

**Issue by SZ-qing** [lazappi/scanpy-in-R#1]
Hi, when I use scanpy in the R environment in jupyterlab, the code is as follows:
sc <- import('scanpy')
sc$pl$umap(adata,color = adata$obs['celltype'])

but there is no figure to show and no file to save. 

I confirm that the packages to be used are installed and loaded,
 how can I deal with this question, and I am very much looking forward to getting your answer.

**Comment by lazappi** [comment]
Hi @nierq 

Is your Jupyter notebook running an R kernal or a Python kernal? Are you able to see other plots or is it just the scanpy plots that aren't displaying?

I will admit that I don't use Jupyter much so I may not be able to help with this.

**Comment by SZ-qing** [comment]
> Hi @nierq
> 
> Is your Jupyter notebook running an R kernal or a Python kernal? Are you able to see other plots or is it just the scanpy plots that aren't displaying?
> 
> I will admit that I don't use Jupyter much so I may not be able to help with this.

Yes , my Jupyter notebook is running an R kernal, and i can see ggplot plots. It doesn't seem to show a plot of scanpy。

**Comment by lazappi** [comment]
It looks as though this might be an issue with using **{reticulate}** and Jupyter https://github.com/rstudio/reticulate/issues/488. I probably can't help much more than that.

You should be able to save the plot to a file using the `save` argument if that's helpful.
